Dockerfile

```diff
@@ -20,7 +20,7 @@ USER 10000:10000
 ENTRYPOINT ["/moco-controller"]
 
 # For MySQL binaries
-FROM --platform=$TARGETPLATFORM ghcr.io/cybozu-go/moco/mysql:8.0.32.1 as mysql
```

Comment:
We have deleted images that had the incorrect tag.
So, the `ghcr.io/cybozu-go/moco/mysql:8.0.32.1` does not exist.